vars/savePodsLogs.groovy

```diff
@@ -0,0 +1,20 @@
+import uk.gov.hmcts.contino.DockerImage
+
+
+def call(DockerImage dockerImage, Map params, String scope) {
```

Comment:
you may want to wrap this all in a `try..catch` (or `catchError(buildResult: 'SUCCESS') {...}` so that errors in this function don't cause

Reply:
Good point. Will do.